Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactions+Types.swift

```diff
@@ -67,3 +67,86 @@ extension ChatMessageReactionsBubbleStyle {
         self == .bigIncoming || self == .smallIncoming
     }
 }
+
+/// Default reactions sorting provided by Stream.
+public enum ReactionSorting {
+    /// Sorting by score.
+    public static func byScore() -> (ChatMessageReactionData, ChatMessageReactionData) -> Bool {
+        { lhs, rhs in
+            lhs.score > rhs.score
+        }
+    }
```

Comment:
Can't we do 
```swift
public static func byScore(_ lhs: ChatMessageReactionData, _: rhs: ChatMessageReactionData) -> Bool {
lhs.score > rhs.score
}
```
and then 
`components.reactionsSorting = ReactionSorting.byScore`
?

Reply:
Yes this would work as well